.github/workflows/publish.yml

```diff
@@ -16,15 +16,15 @@ jobs:
         run: sh .github/scripts/check-release.sh
       - name: Install dependencies
         run: yarn && yarn install:functions
-      - name: Enter functions directory
-        run: cd functions
       - name: Publish with latest tag
         if: "!github.event.release.prerelease"
-        run: npm publish .
+        run:
+          npm publish functions
```

Comment:
You could use `working-directory` once in the job definition, and all the commands will use it :)

Reply:
Yes but it is for all the jobs in the same step and the `Check release validity` step need to be at the root